docs/migration-guides/sfa-ios-v7-to-v8.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: SFA iOS SDK - v7 to v8
+description: "SFA iOS SDK - v7 to v8 | Documentation - Web3Auth"
+sidebar_label: v7 to v8
+---
+
+This migration guide provides steps for upgrading from version v7 to v8 of the SFA iOS SDK. The
+guide outlines significant changes and enhancements.
+
+## Breaking Changes
+
+### Changes in SingleFactorAuth Constructor
+
+In v8, we try to improve the developer experience by renaming the parameter and
+`SingleFactorAuthArgs` class, and introducing a new parameter for session management.
+
+- `singleFactorAuthArgs` used to initialize the SingleFactorAuth SDK has been renamed to
+  `sfaParams`. It has been renamed to align the naming conventions across Web3Auth SDKs.
+
+#### SingleFactorAuthArgs Changes
+
+- `SingleFactorAuthArgs` is now renamed to `SFAParams`. It has been renamed to align with naming
+  conventions across Web3Auth SDKs.
+- `sessionTime` was added to specify session duration in seconds. By default, the value is set to
+  86400 seconds (1 day), with a maximum session duration of up to 7 days
+
+```swift
+// remove-next-line
+let arguments = SingleFactorAuthArgs(
+// add-next-line
+let singleFactorAuthArgs = SFAParams(
+    web3AuthClientId: "<Your Client Id>",
```

Comment:
`<Your Client Id>`, let's write these as `YOUR_WEB3AUTH_CLIENT_ID` across the docs.